icecreamshop/icecreamshop/src/main/java/com/summerschool/icecreamshop/repository/ProductRepository.java

```diff
@@ -8,4 +8,5 @@
 @Repository
 public interface ProductRepository extends JpaRepository<Product, Long> {
 
+    Long getId();
```

Comment:
Aici trebuie creata o metoda care sa iti salveze produsul in baza de date, sau sa le folosesti pe cele deja prestabilite